pandas/tests/io/excel/test_xlrd.py

```diff
@@ -1,35 +1,29 @@
-import pandas.util._test_decorators as td
+import pytest
 
 import pandas as pd
 import pandas.util.testing as tm
 from pandas.util.testing import ensure_clean
 
 from pandas.io.excel import ExcelFile
 
+xlrd = pytest.importorskip("xlrd")
+xlwt = pytest.importorskip("xlwt")
```

Comment:
Do we already have a file we can use that contains the content of `frame`? Having to include xlwt creates an unnecessary dependency in this module